.travis.yml

```diff
@@ -281,6 +281,11 @@ install:
 script:
   # Don't run doctr if the build fails
   - set -e
+  - if [[ "${TEST_SPHINX}" == "true" ]]; then
+        cp bin/changelog.py ~/changelog.py;
+        chmod +x ~/changelog.py;
```

Comment:
Why is the copy necessary? And you should be able to set the permissions to executable in git.

Reply:
@asmeurer AFAIK, doctr runs command in the deploy repo, not build repo, and files in the build repo get stashed.

Reply:
Oh yeah, because it's assumed that any command in the main repo you could just run yourself before running doctr. So I guess this *is* the correct thing to do. We should document this in the doctr docs. I hadn't thought this use-case before (command to run in the deploy repo that lives in the build repo).